services/scan/src/server.ts

```diff
@@ -162,6 +162,21 @@ export function buildApp({ store, importer }: AppOptions): Application {
   app.delete<NoParams, DeleteElectionConfigResponse>(
     '/config/election',
     async (_request, response) => {
+      const batches = await store.batchStatus();
+      if (!batches.every((b) => b.exportedAt)) {
```

Comment:
This construct is used a few times in this PR. Consider making something equivalent to this a method on `Store`.

Reply:
I've added `Store.getCanUnconfigure()`, and then also added `canUnconfigure` to `GetScanStatusResponse`.